src/mlpack/bindings/python/print_input_processing.hpp

```diff
@@ -54,35 +55,132 @@ void PrintInputProcessing(
    *
    * # Detect if the parameter was passed; set if so.
    * if param_name is not None:
+   *   if isinstance(param_name, int):
    *   SetParam[int](<const string> 'param_name', param_name)
    *   CLI.SetPassed(<const string> 'param_name')
```

Comment:
Oops, for this comment to be correct Python we should indent this bit.  (I'm being picky about this comment because this code below is really hard to read otherwise.  I don't think there's much we can do about that though. :))